Compiling scss error stops watch process

Whenever there's an error compiling my scss file, the whole process stops with an exit code. 

I get the following output:  

Error: error /Users/clara/Documents/tests/project/client/assets/scss/_base.scss (Line 357: Undefined variable: "$global-radius".)

npm ERR! Darwin 13.4.0
npm ERR! argv "node" "/usr/local/bin/npm" "start"
npm ERR! node v0.10.29
npm ERR! npm  v2.1.3
npm ERR! code ELIFECYCLE
npm ERR! foundation-apps-template@0.0.1 start: `gulp`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the foundation-apps-template@0.0.1 start script.
npm ERR! This is most likely a problem with the foundation-apps-template package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp
npm ERR! You can get their info via:
npm ERR!     npm owner ls foundation-apps-template
npm ERR! There is likely additional logging output above.

BTW, $global-radius is a variable defined on the _settings.scss file. Do I need to uncomment it in order to use it? No need for it on Foundation so I'm a bit confused.

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -76,6 +76,9 @@ gulp.task('sass', function() {
       style: 'nested',
       bundleExec: true
     }))
+    .on('error', function(e) {
+      console.log(e);
+    })
     .pipe(autoprefixer({
       browsers: ['last 2 versions', 'ie 10']
     }))

